pykanto/dataset.py

```diff
@@ -749,7 +749,7 @@ def write_to_json(self) -> None:
         save_to_jsons(self)
 
     @timing
-    def cluster_ids(self, min_sample: int = 10) -> None:
+    def cluster_ids(self, min_sample: int = 10, **kwargs) -> None:
```

Comment:
You would need to pass different kwargs to `umap_reduce` and `hdbscan_cluster`: the current changes pass all kwargs to both functions, which leads to conflicts/errors. See:

- https://gist.github.com/pykong/6a9b020e4df2f56f23e89f15733dbc4b
- https://colabdoge.medium.com/dealing-with-multiple-kwargs-in-python-6babd7be531c
- https://stackoverflow.com/questions/71713742/workaround-for-multiple-parameters-are-not-allowed